### PR TITLE
Reorganize settings section

### PR DESCRIPTION
**Pull request by numahell**
Reorganize settings section.

## Description

- Reorganize advanced settings with fewer sections but more visible sub-sections : https://geotrek.readthedocs.io/en/docs-reorg-settings-refsettings-in-advanced/install/advanced-configuration.html
- Merge settings details in appropriate parts of advanced configuration page
- Move basic settings in Configuration page
- Keep "settings details" section title to avoid URL redirection

## Related Issue

Discussion started on this PR #3669.

## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/CONTRIBUTING.html)
- [x] My code respects the Definition of done available in the [Development section of the documentation]()https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields
- [x] I have performed a self-review of my code
- ~~I have commented my code, particularly in hard-to-understand areas~~
- ~~I have made corresponding changes to the documentation~~
- ~~I have added tests that prove my fix is effective or that my feature works.~~
- ~~New and existing unit tests pass locally with my changes~~
- [x] I added an entry in the changelog file
- ~~My commits are all using prefix convention (emoji + tag name) and references associated issues~~
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [x] The title of my PR mentionned the issue associated

### CODE REVIEW

**Review by babastienne — APPROVED** [review]

File: docs/changelog.rst [hunk 1]
```diff
@@ -16,6 +16,7 @@ CHANGELOG
 - Update documentation fr translation files
 - Uniformize documentation section
 - Move user management section to user manual (#3709)
+- Reorganize settings section (related to PR #3669)
```
Comment (babastienne) [inline docs/changelog.rst:19]:
```suggestion
- Reorganize settings section (related to issue #3669)
```

**Review by babastienne — CHANGES_REQUESTED** [review]
For your commits try to squash them all in order to reduce the number of commits. also you can mention related issue in the commit message. If you add "fix #3669" in your commit message the issue will automatically be closed when merging the pull request. If you don't want the issue to be automatically closed you can just add "ref #3669" to reference the issue.

File: docs/changelog.rst (hunk 1 above)
Comment (numahell) [reply docs/changelog.rst:24]: Oui mais c'est en fait une PR, même si c'est le brouillon de celle-ci et qu'elle ne sera jamain mergée je voulais être précise.

Comment (babastienne) [reply docs/changelog.rst:24]: Ah ok my bad :laughing:

**Review by babastienne — APPROVED** [review]

**Pull request merged**